packages/flutter_tools/test/general.shard/hot_test.dart

```diff
@@ -212,12 +188,9 @@ void main() {
         fileSystem.file('.packages')
           ..createSync(recursive: true)
           ..writeAsStringSync('\n');
-        final MockDevice mockDevice = MockDevice();
-        when(mockDevice.supportsHotReload).thenReturn(true);
-        when(mockDevice.supportsHotRestart).thenReturn(true);
-        when(mockDevice.supportsFlutterExit).thenReturn(false);
```

Comment:
It looks like this is hard-coded to true in `FakeDevice`. Was this being false not needed for this test?

Reply:
It was not